README.md

```diff
@@ -95,6 +95,34 @@ authoring doc comments for Dart with `dartdoc`.
 
 ## Advanced features
 
+### dartdoc_options.yaml
+
+Creating a file named dartdoc_options.yaml at the top of your package can change how Dartdoc
+generates docs.  
+
+```yaml
+dartdoc:
```

Comment:
Why have a `dartdoc` top key in `dartdoc_options.yaml`?

Couldn't `categoryOrder` just be at the top?

Reply:
This was paralleling with `analysis_options.yaml`, which has analyzer and linter subsections.  There might be an opportunity for sections for other tools using dartdoc libraries, such as the still vaporware API comparison gadget.